utilities/infra.py

```
@@ -1011,6 +1012,23 @@ def login_to_account(login_command):
         return False
 
 
+def get_oc_whoami_username(*, wait_timeout: int = TIMEOUT_30SEC, sleep: int = 3) -> str:
+    """Return the current OpenShift user from ``oc whoami``, retrying until the command succeeds."""
+
+    def _whoami() -> str:
+        return check_output(["oc", "whoami"]).decode().strip()
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)